### PR TITLE
Capitalization Fixes

### PR DESCRIPTION
**Pull request by mepland**
Also refactored missed `_format_axes()` in `interpretation.py` and `classifiers.py`

### CODE REVIEW

**Comment by mepland** [comment]
@parrt I don't know why these lines are included to change the x-axis ticks of the importance plots, but I think they should be removed.

Before:
```
    # TODO this just screws up the x-axis ticks, should just leave numeric!
    ax.set_xticks(range(0, len(shadow_tree.feature_names)))
    ax.set_xticklabels(shadow_tree.feature_names)
```
image

After (https://github.com/parrt/dtreeviz/pull/245/commits/73462af699cfa0f8dc6542af5e7a847d72488287):
image

**Comment by parrt** [comment]
> I don't know why these lines are included to change the x-axis ticks of the importance plots, but I think they should be removed.

Agreed. I believe I recently change this to be horizontal and probably left some code which is strange but yes all of the feature names should be on the vertical axis now.

**Comment by parrt** [comment]
> I looked through the documentation for sklearn and spark trees and pulled all of the criterion values I could find.

Excellent

File: dtreeviz/models/sklearn_decision_trees.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ def get_features(self):
         return self.tree_model.tree_.feature
 
     def criterion(self):
-        return self.tree_model.criterion.upper()
+        return criterion_remapping.get(self.tree_model.criterion, self.tree_model.criterion)
```
Comment (parrt) [inline dtreeviz/models/sklearn_decision_trees.py:40]: I wonder if we should just make a function that automatically gets the default value to hide the double reference to the criterion in the function call.

File: dtreeviz/models/spark_decision_tree.py [hunk 2]
```diff
@@ -99,7 +100,7 @@ def get_features(self) -> np.ndarray:
         return self.features
 
     def criterion(self) -> str:
-        return self._get_tree_model_parameter_value("impurity")
+        return criterion_remapping.get(self._get_tree_model_parameter_value("impurity"), self._get_tree_model_parameter_value("impurity"))
```
Comment (parrt) [inline dtreeviz/models/spark_decision_tree.py:103]: A small issue but might be better to simplify this into a function call. As above.

File: dtreeviz/models/sklearn_decision_trees.py (hunk 1 above)
Comment (mepland) [reply dtreeviz/models/sklearn_decision_trees.py]: Done! https://github.com/parrt/dtreeviz/pull/245/commits/c0f6a0fe6b620bd85a4a8a3f0fd61c68271e866b

**Pull request merged**